lib/instrumentation/grpc-js/nr-hooks.js

```diff
@@ -13,16 +13,19 @@ module.exports = [
   {
     type: 'generic',
     moduleName: '@grpc/grpc-js/build/src/resolving-call',
+    isEsm: true,
```

Comment:
Wait, why did this have to get added? It's not an ESM module, and I thought that's what the flag is for?

Reply:
This code is getting transpiled and it has named exports instead of default, that's why